src/dmd/globals.d

```diff
@@ -362,6 +362,44 @@ struct Global
         _version = (import("VERSION") ~ '\0').ptr;
         compiler.vendor = "Digital Mars D";
     }
+
+    /**
+    Returns: the version as the number that would be returned for __VERSION__
+    */
+    extern(C++) uint versionNumber()
+    {
+        import core.stdc.ctype;
+        static uint cached = 0;
```

Comment:
Don't use thread local storage for dmd. Use `__gshared`

Reply:
good catch